hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/HoodieHiveUtils.java

```diff
@@ -146,4 +154,74 @@ public static List<String> getIncrementalTableNames(JobContext job) {
   public static boolean isIncrementalUseDatabase(Configuration conf) {
     return conf.getBoolean(HOODIE_INCREMENTAL_USE_DATABASE, false);
   }
+
+  public static boolean SUPPORT_TIMESTAMP_WRITEABLE_V2;
+  private static Class timestampClass = null;
+  private static Method setTimeInMillis = null;
+  private static Constructor timestampWriteableV2constructor = null;
+
+  public static boolean SUPPORT_DATE_WRITEABLE_V2;
+  private static Constructor dateWriteableV2constructor = null;
+
+  static {
+    // timestamp
+    try {
+      timestampClass = Class.forName("org.apache.hadoop.hive.common.type.Timestamp");
+      setTimeInMillis = timestampClass.getDeclaredMethod("setTimeInMillis", long.class);
+      Class twV2Class = Class.forName("org.apache.hadoop.hive.serde2.io.TimestampWritableV2");
+      timestampWriteableV2constructor = twV2Class.getConstructor(timestampClass);
```

Comment:
Pls make those className as static final constant

Reply:
oh sorry,   i mean we can make "org.apache.hadoop.hive.serde2.io.TimestampWritableV2", "org.apache.hadoop.hive.common.type.Timestamp"  as constant string thanks